go/vt/schemadiff/schema.go

```diff
@@ -134,21 +134,31 @@ func getForeignKeyParentTableNames(createTable *sqlparser.CreateTable) (names []
 }
 
 // getViewDependentTableNames analyzes a CREATE VIEW definition and extracts all tables/views read by this view
-func getViewDependentTableNames(createView *sqlparser.CreateView) (names []string) {
+func getViewDependentTableNames(createView *sqlparser.CreateView) (names []string, cteNames []string) {
+	cteMap := make(map[string]bool)
 	_ = sqlparser.Walk(func(node sqlparser.SQLNode) (kontinue bool, err error) {
 		switch node := node.(type) {
+		case *sqlparser.CommonTableExpr:
+			if !cteMap[node.ID.String()] {
```

Comment:
does this handle case sensitivity correctly?

Reply:
Fixed and tested.